dataactbroker/handlers/accountHandler.py

```diff
@@ -479,8 +514,8 @@ def isAccountExpired(self, user):
         """
         today = parse(time.strftime("%c"))
         daysActive = (today-user.last_login_date).days
-        secondsActive = (today-user.last_login_date).seconds
-        if daysActive > self.INACTIVITY_THRESHOLD or (daysActive == self.INACTIVITY_THRESHOLD and secondsActive > 0):
+        # secondsActive = (today-user.last_login_date).seconds
```

Comment:
We should probably remove this comment so future team members don't get confused by the glory days of when we tracked active accounts by the second.